src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const pricePerDay = 40;
+
+  if (days >= 7) {
```

Comment:
Please, don't use numbers in conditions, create variables.
For example:

 const LONG_TERM = 7;